CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ For older changes see the [archived Singularity change log](https://github.com/a
   but `fakeroot-sysv` is much faster.
 - Fix the `--rocm` flag in combination with `-c` / `-C` by forwarding all
   `/dri/render*` devices into the container.
+- Fix the locating of shared libraries when running `unsquashfs` from a
+  non-standard location the way conda does.
+- Fix the creation of missing bind points when using image binding with
+  underlay.
```

Comment:
These are duplicates, a merge error.

Reply:
this part is corrupted during the cherry-pick. I only copy the newly added description change and append it to the parent branch.